### PR TITLE
[Homework #18] - React UI by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

File: homeworks/ivan-ishchenko/18-react-ui/style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.avatars {
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/18-react-ui/style.css:1]: Please fix styles according to the Figma file.
1. Reduce sizes of images.
2. 5 avatars in row.
3. Refresh all buttons shouldn't be fixed. Fix the mobile screen as well.

Screenshot 2023-09-26 at 13 51 41

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
Hello @ViVse!
I have checked your PR and left some comments.

**Comment by ViVse** [comment]
@VitalikMelnyk, fixed all problems.

**Review by VitalikMelnyk — APPROVED** [review]
Approved.

**Pull request merged**